### PR TITLE
Add a special upgrade notice for 1.7.5

### PR DESCRIPTION
**Pull request by linsun**
given 1.7.5 is released, @brian-avery suggested to edit the file in istio.io directly.
[ ] Configuration Infrastructure
[ x ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/en/news/releases/1.7.x/announcing-1.7.5/index.md [hunk 1]
```diff
@@ -31,4 +31,4 @@ This release contains bug fixes to improve robustness. This release note describ
 
 ### Upgrade Notice
 
-When upgrading your Istio data plane from 1.7.x (where x < 5) to 1.7.5, you may observe connectivity issues between your gateway to your sidecars or among your sidecars with 503 errors in the log. This happens when some of your proxies are on 1.7.x while some are on 1.7.5, and 1.7.x proxies could not properly process any 1xx or 204 response code from 1.7.5 proxies. To fix this, upgrade all your proxies(gateways and sidecars) to 1.7.5 as soon as possible. ([Issue 29427](https://github.com/istio/istio/issues/29427)
+When upgrading your Istio data plane from 1.7.x (where x < 5) to 1.7.5 or newer, you may observe connectivity issues between your gateway to your sidecars or among your sidecars with 503 errors in the log. This happens when some of your proxies are on 1.7.x while some are on 1.7.5, and 1.7.x proxies could not properly process any 1xx or 204 response code from 1.7.5 proxies. To fix this, upgrade all your proxies(gateways and sidecars) to 1.7.5 as soon as possible. ([Issue 29427](https://github.com/istio/istio/issues/29427)
```
Comment (Monkeyanator) [inline content/en/news/releases/1.7.x/announcing-1.7.5/index.md:34]:
```suggestion
When upgrading your Istio data plane from 1.7.x (where x < 5) to 1.7.5 or newer, you may observe connectivity issues between your gateway and your sidecars or among your sidecars with 503 errors in the log. This happens when 1.7.5+ proxies send HTTP 1xx or 204 response codes with headers that 1.7.x proxies reject. To fix this, upgrade all your proxies (gateways and sidecars) to 1.7.5 as soon as possible. ([Issue 29427](https://github.com/istio/istio/issues/29427), [More information](https://github.com/istio/istio/pull/28450)) 
```

**Review by Monkeyanator — COMMENTED** [review]
Looks great, just a few minor edits. Might need to add similar change to 1.8.x upgrade notes about upgrading from 1.7.4-.

**Comment by linsun** [comment]
> Looks great, just a few minor edits. Might need to add similar change to 1.8.x upgrade notes about upgrading from 1.7.4-.

Thanks Sam!  Updated the 1.8 notice as well - https://github.com/istio/istio/pull/29484

**Review by Monkeyanator — APPROVED** [review]

**Review by brian-avery — APPROVED** [review]

File: content/en/news/releases/1.7.x/announcing-1.7.5/index.md [hunk 2]
```diff
@@ -28,3 +28,7 @@ This release contains bug fixes to improve robustness. This release note describ
...
 
 - **Fixed** missing listeners on a VM when the VM sidecar is connected to `istiod` but a `WorkloadEntry` is registered later. ([Issue #28743](https://github.com/istio/istio/issues/28743))
+
+### Upgrade Notice
+
+When upgrading your Istio data plane from 1.7.x (where x < 5) to 1.7.5 or newer, you may observe connectivity issues between your gateway and your sidecars or among your sidecars with 503 errors in the log. This happens when 1.7.5+ proxies send HTTP 1xx or 204 response codes with headers that 1.7.x proxies reject. To fix this, upgrade all your proxies (gateways and sidecars) to 1.7.5+ as soon as possible. ([Issue 29427](https://github.com/istio/istio/issues/29427), [More information](https://github.com/istio/istio/pull/28450))
```
Comment (howardjohn) [inline content/en/news/releases/1.7.x/announcing-1.7.5/index.md:34]: Is this considered a bug or working as intended? Do we have plans to address this?

Comment (linsun) [reply content/en/news/releases/1.7.x/announcing-1.7.5/index.md:34]: My understanding is this is how it is until it is addressed in envoy - https://github.com/envoyproxy/envoy/issues/13868

cc @Monkeyanator to correct me if I'm wrong.

Comment (Monkeyanator) [reply content/en/news/releases/1.7.x/announcing-1.7.5/index.md:34]: You're correct. Think we're stuck with this issue on upgrades from 1.7.4- to all newer versions without an envoy fix

**Review by frankbu — APPROVED** [review]

File: content/en/news/releases/1.7.x/announcing-1.7.5/index.md (hunk 2 above)
Comment (rvennam) [inline content/en/news/releases/1.7.x/announcing-1.7.5/index.md:34]: Just a suggestion for clarity: "This happens when 1.7.5+ proxies send HTTP 1xx or 204 response codes with headers that 1.7.x proxies reject." -> "This happens when 1.7.5+ proxies send HTTP 1xx or 204 response codes with headers that 1.7.4- proxies and gateways reject."

Comment (linsun) [reply content/en/news/releases/1.7.x/announcing-1.7.5/index.md:34]: gateway is not a sidecar but also considered as proxies. :) -- proxies (gateways and sidecars)

**Pull request merged**